### ISSUE TITLE
Code Explorer should allow node drag-and-drop copy to other projects

### ISSUE THREAD

**Issue by ThunderFrame** [rubberduck-vba/Rubberduck#1815]
Just like Project Explorer allows drag'n'drop copy of:
- Standard Modules
- Class Modules
- UserForms

But Code Explorer could also offer:
- Folders
- Members

Would be a nice bonus if multiple items could be copied at once.

**Comment by Hosch250** [comment]
What do you mean by members?  Like, drag/drop a procedure to another file?  This is possible, but could be a wee bit tricky, so I'm probably not going to do it until after we get a stable 2.0 out.  Multiple items?  We don't even enable selection of multiple items, so I think I won't support that.

**Comment by ThunderFrame** [comment]
Yeah, by members, I mean dragging a procedure, field or constant. But given there might be dependencies, I guess such a copy would need to be a bit like a refactoring, and take account of the dependent declarations.

Primary focus should be on copying modules/classes and forms. They might also have dependencies, but matching Project Explorer capabilities should be the initial objective.

**Comment by retailcoder** [comment]
CE needs a "Copy" command for all these node types first; any drag-and-drop functionality for it would be built on top of the same context menu commands.
